Makefile

```diff
@@ -47,11 +47,11 @@ escript:
 
 .PHONY: gradualize
 gradualize: escript
-	./gradualizer -pa src/ -I include src/*.erl
+	./gradualizer -pa src/ -I include -- src/*.erl
```

Comment:
`--` is not really needed here until we allow multiple dirs for `-I`. Perhaps it's still good to use `--`, for anyone looking at the Makefile for inspiration.

Reply:
yes, it's not necessary but exactly as you say. and I wanted to make it future proof  if we happen to allow multiple dirs in include too.